src/main/kotlin/no/digdir/catalog_admin_service/model/CodeList.kt

```diff
@@ -11,5 +11,11 @@ data class CodeList(
     val catalogId: String,
     val description: String,
     val codes: List<Code>
+)
 
+data class CodeListToBeCreated(
```

Comment:
Med dette så vil den ikke feile om noen legger til felter i body som ikke finnes i `CodeListToBeCreated`, feks `id` og `catalogId`
```suggestion
@JsonIgnoreProperties(ignoreUnknown = true)
data class CodeListToBeCreated(
```